pkg/registry/networking/networkpolicy/strategy.go

```diff
@@ -26,6 +26,9 @@ import (
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	"k8s.io/kubernetes/pkg/apis/networking"
 	"k8s.io/kubernetes/pkg/apis/networking/validation"
+	"k8s.io/kubernetes/pkg/features"
+
```

Comment:
why the line break here?

Reply:
I'll now blame everything into my lazy vscode config ;x